freqtrade/exchange/exchange.py

```diff
@@ -139,21 +139,9 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-class Exchange:
+class BaseExchange:
```

Comment:
I think this should move to it's own file - that would make the code easier to read and refind.
(i hate to scroll to line 200 to find the class-definition ...)

It's fine to have small classes in one file i guess - but this file is over 1000 lines long - so every line removed will help :D